ui/pages/confirmations/components/multilayer-fee-message/multi-layer-fee-message.js

```diff
@@ -19,50 +17,27 @@ export default function MultilayerFeeMessage({
   plainStyle,
 }) {
   const t = useContext(I18nContext);
-  const [fetchedLayer1Total, setLayer1Total] = useState(null);
+  const { layer1GasFee } = transaction;
 
   const useCurrencyRateCheck = useSelector(getUseCurrencyRateCheck);
 
-  useEffect(() => {
-    if (!transaction?.txParams) {
-      return;
-    }
-    const getEstimatedL1Fee = async () => {
-      try {
-        const result = await fetchEstimatedL1Fee(
-          transaction?.chainId,
-          transaction,
-        );
-        setLayer1Total(result);
-      } catch (e) {
-        captureException(e);
```

Comment:
Unfortunately we do lose any specific error tracking if the flow fails, but we could add suitable properties to the metadata if it becomes an issue.

For example, we could have an `layer1Fee: {value?: Hex, error?: string}`.